packages/panzer/mini-em/example/BlockPrec/main.cpp

```diff
@@ -619,12 +619,8 @@ void createExodusFile(const std::vector<Teuchos::RCP<panzer::PhysicsBlock> >& ph
     }
 
     Teuchos::ParameterList output_pl("Output");
-    Teuchos::ParameterList& cell_avg_q = output_pl.sublist("Cell Average Quantities");
```

Comment:
These lines are actually all necessary because they ensure that these sublists exist. We really just need empty sublists called "Cell Average Quantities", "Cell Quantities", "Nodal Quantities", and "Allocate Nodal Quantities".

Reply:
I just undid the changes to `BlockPrec/main.cpp`.